devices/src/acpi.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright © 2019 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use vmm_sys_util::eventfd::EventFd;
+use BusDevice;
+
+/// A device for handling ACPI shutdown and reboot
+pub struct AcpiDevice {
```

Comment:
I would rename this as `AcpiShutdownDevice` or something similar to make sure we don't think this is some sort of generic ACPI device.